packages/resolve-scripts/src/configs/webpack.client.config.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
 
     plugins: [
         new webpack.DefinePlugin({
+            'process.env': JSON.stringify(process.env),
```

Comment:
Why are the both variants used  (process.env and process.env.NODE_ENV)? As I understand, the only process.env will be enough